Makefile

```diff
@@ -16,17 +16,21 @@ CMDS=smbplugin
 PKG = github.com/kubernetes-csi/csi-driver-smb
 GINKGO_FLAGS = -ginkgo.v -ginkgo.timeout=2h
 GO111MODULE = on
-GOPATH ?= $(shell go env GOPATH)
+ifndef GOPATH
+GOPATH := $(shell go env GOPATH)
+endif
 GOBIN ?= $(GOPATH)/bin
 DOCKER_CLI_EXPERIMENTAL = enabled
 IMAGENAME ?= smb-csi
 export GOPATH GOBIN GO111MODULE DOCKER_CLI_EXPERIMENTAL
 
 include release-tools/build.make
 
-GIT_COMMIT ?= $(shell git rev-parse HEAD)
+ifndef GIT_COMMIT
+GIT_COMMIT := $(shell git rev-parse HEAD)

```

Comment:
`GIT_COMMIT ?=`  sets the value of the GIT_COMMIT variable to the SHA1 hash of the current commit, if it is not already set, so you don't need to check `ifndef GIT_COMMIT`